lib/web/users.go

```diff
@@ -219,9 +241,15 @@ type userAPIGetter interface {
 }
 
 type saveUserRequest struct {
-	Name   string   `json:"name"`
-	Roles  []string `json:"roles"`
-	Logins []string `json:"logins,omitempty"`
+	Name          string    `json:"name"`
+	Roles         []string  `json:"roles"`
+	Logins        *[]string `json:"logins,omitempty"`
```

Comment:
Let's add a comment and mention the difference between passing `nil` and passing an empty slice (which, I assume, means we'll clear the logins). Let's also spell out if some fields are only supported on creation or update.

It's worth mentioning that all the fields below work the same.